src/libstore/sandbox-defaults.sb

```diff
@@ -98,7 +98,9 @@
 (allow file*
        (literal "/private/var/select/sh"))
 
-; Allow Rosetta 2 to run x86_64 binaries on aarch64-darwin.
+; Allow Rosetta 2 to run x86_64 binaries on aarch64-darwin (and vice versa).
```

Comment:
> (and vice versa)

Rosetta 2 can't be run on x86_64 systems, maybe this comment should be updated to be more accurate?